test/conformance/testdata/conformance.txt

```diff
@@ -26,6 +26,7 @@ test/e2e/apps/deployment.go: "deployment should support proportional scaling"
 test/e2e/apps/rc.go: "should serve a basic image on each replica with a public image"
 test/e2e/apps/rc.go: "should adopt matching pods on creation"
 test/e2e/apps/rc.go: "should release no longer matching pods"
+test/e2e/apps/rc.go: "should surface a failure condition on a common issue like exceeded quota"
```

Comment:
this was added by hand so it's failing the check, please see the diff to know where to put it, or follow the instructions I linked to

Reply:
Thanks for the review. Commit rebuilt and refreshed as per the expected norm